.babelrc

```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
+    "test": {
+      "plugins": ["istanbul"]
+    }
+  },
+  "ignore": ["node_modules/**/*"],
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-transform-flow-strip-types",
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-export-default-from",
+    "@babel/plugin-transform-property-mutators",
+    "@babel/plugin-proposal-object-rest-spread",
+    [
+      "@babel/plugin-transform-spread",
```

Comment:
what is that one for? it does not exist on the other plugins or UI

Reply:
removed